tests/unit/test_mixins.py

```diff
@@ -51,19 +52,6 @@ def children(self):
         return None
 
 
-@ddt.ddt
-class CompletionMixinTestCase(TestCase):
-
-    @ddt.data(*app_config.PROGRESS_DETACHED_CATEGORIES)
-    def test_all_blocks_excluded_from_completion(self, blockclass):
-        xblock = XBlock.load_class(blockclass)
-        self.assertEqual(
-            XBlockCompletionMode.get_mode(xblock),
-            XBlockCompletionMode.EXCLUDED,
-            "Block {!r} did not have completion mode 'excluded'".format(xblock),
-        )
-
-
 class TestChildrenNavigationXBlockMixin(TestWithPatchesMixin, TestCase):
```

Comment:
Actually, let's restore this code, but move `PROGRESS_DETACHED_CATEGORIES` to this file and call it:
`PROGRESS_EXCLUDED_BLOCKS`